CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased (patch)
```

Comment:
We don't have the changeset config here, so added a proposal for the Changelog.

Please let me know if this is something I should avoid 🙇

Reply:
You could follow https://keepachangelog.com/en/1.1.0/

i.e. `## [Unreleased]`

Reply:
We don't need a strict rule here. 😃